haero/surface.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021, National Technology & Engineering Solutions of Sandia,
+// LLC (NTESS). Copyright (c) 2022, Battelle Memorial Institute
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifndef HAERO_SURFACE_HPP
+#define HAERO_SURFACE_HPP
+
+#include <haero/haero.hpp>
+
+#include <ekat/ekat_assert.hpp>
+
+namespace haero {
+
+class Surface final {
+
+  // use only for creating containers of Surface related variables!
+public:
+  Surface() = default;
+
+  // these are supported for initializing containers of Surface
+  Surface(const Surface &rhs) = default;
+  Surface &operator=(const Surface &rhs) = default;
+
+  /// destructor, valid on both host and device
+  KOKKOS_FUNCTION
+  ~Surface() {}
+
+  // land fraction [unitless]
+  Real landfrac = 0.0;
```

Comment:
I think the initializations can be omitted here in favor of a default constructor that zeros all the fields.

Reply:
I removed these initializations, in favor of  `Surface s{};` where relevant in the testing.